File: GILua/src/main/java/org/anime_game_servers/gi_lua/models/scene/group/CrucibleConfig.java
package org.anime_game_servers.gi_lua.models.scene.group;

import lombok.Getter;
import org.anime_game_servers.core.base.annotations.lua.LuaNames;

import java.util.List;

@Getter
public class CrucibleConfig {
    private int duration;
    @LuaNames("start_cd")
    private int startCd;
    @LuaNames("mp_play_id")
    private int mpPlayId;
    @LuaNames("mp_play_id_list")
    private List<Integer> mpPlayIdList;
}

File: GILua/src/main/java/org/anime_game_servers/gi_lua/models/scene/group/CrucibleConfig.kt
package org.anime_game_servers.gi_lua.models.scene.group

import org.anime_game_servers.core.base.annotations.lua.LuaNames

data class CrucibleConfig (
    val duration: Int = 0,
    @field:LuaNames("start_cd")
    val startCd: Int = 0,
    @field:LuaNames("mp_play_id")
    val mpPlayId: Int = 0,
    @field:LuaNames("progress_stage")
    val progressStageList: List<Int>? = null
)
